lib/Core/Executor.cpp

```diff
@@ -50,6 +49,7 @@
 #include "klee/Internal/System/MemoryUsage.h"
 #include "klee/SolverStats.h"
 
+#include "../Expr/ArrayExprOptimizer.h"
```

Comment:
I thought relative paths are discouraged, no?

Reply:
yes they are. but i felt that this optimisation is a specific implementation detail. Therefore, it's nicer to have it closer to the actual implementation instead of being part of the project `include` directory - a location for more general implementation.

One way to fix that in the future might be to add a general optimiser class which can be extended arbitrarily (i.e. by the ArrayExprOptimizer).

Reply:
I see.  OK, let's leave it like this